### DISCUSSION TITLE
Project Download fails

### DISCUSSION THREAD

**Discussion by MonikaShah** [opengisch/QField#3724] [Bugs]
Please help in resolving
![Screenshot from 2022-12-07 14-02-47](https://user-images.githubusercontent.com/4188149/206128441-c2ad4cac-f860-48c7-9b4c-414b5671682a.png)

**Comment by suricactus** [comment]
There was a demporary downgrade of our services. All should be up and running now.

**Comment by MonikaShah** [comment]
Dear sir,
I am continously facing the same issue . Today also i created a new project
for  SWM and faced the issue saying
08:29:39.156 QGSSTDERR WARNING QFSFileEngine::open: No file name specified
08:29:39.156 QGSSTDERR INFO Stopping QGIS app….

This is continously error and the project fails to download.
Please explain me what mistake I am making from my side or is it Qfield
cloud issue?

On Wed, Dec 7, 2022 at 7:16 PM Ivan Ivanov ***@***.***> wrote:

> There was a demporary downgrade of our services. All should be up and
> running now.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/opengisch/QField/discussions/3724#discussioncomment-4333209>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA76P5NZNCDMJKKFLPTIR33WMCIJZANCNFSM6AAAAAASWQIVJM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
With Regards,
Monika Shah
